qml/PolarGrid.qml

```diff
@@ -34,9 +36,9 @@ Item {
         delegate: Text {
             font.pixelSize: 15
             // Calculate the angle and radius for each item
-            property var itemRadian: (index%Math.floor(360/repeater.angle))*repeater.angle*Math.PI/180
-            property var itemRadius: repeater.radius*(1 + Math.floor(index/Math.floor(360/repeater.angle)))/repeater.numberOfRings
-            text: repeater.formatDistance(root.maxDistance*(1 + Math.floor(index/Math.floor(360/repeater.angle)))/repeater.numberOfRings)
+            property var itemRadian: ((root.angle/2)%90)*Math.PI/180 - ((root.angle%180) ? Math.PI/2 : 0)
```

Comment:
Some of the math in this file is pretty complicated to understand what/why you are doing. Some comments in the future will help new eyes :eyes:.